rosidl_runtime_py/rosidl_runtime_py/convert.py

```diff
@@ -156,3 +156,14 @@ def _convert_value(value, truncate_length=None):
         # Assuming value is a message since it is neither a collection nor a primitive type
         value = message_to_ordereddict(value, truncate_length=truncate_length)
     return value
+
+
+def slot_types_dict_from_message(msg: Any) -> OrderedDict:
+    """
+    Return an OrderedDict of the slot types of a message.
+
+    :param msg: The ROS message to convert.
+        This does not truncate the list of message fields.
```

Comment:
I am not sure what this line is supposed to mean?

Reply:
This was a copy-paste from other functions docblock, updated in https://github.com/ros2/rosidl_python/pull/33/commits/934aed2408f630d56b864626249e9a0afbed8e52